Week2task1/src/ComplexNumber.java

```diff
@@ -0,0 +1,79 @@
+
+public class ComplexNumber {
+	Integer real;
+	Integer im;
+	/**
+	 * @param real
+	 * @param im
+	 */
+	public ComplexNumber(Integer real, Integer im) {
+		super();
+		this.real = real;
+		this.im = im;
+	}
+
+	public ComplexNumber() {
+		// TODO Auto-generated constructor stub
+	
+	}
+	
+	public Integer getReal() {
+		return real;
+	}
+
+	public void setReal(Integer real) {
+		this.real = real;
+	}
+
+	public Integer getIm() {
+		return im;
+	}
+
+	public void setIm(Integer im) {
+		this.im = im;
+	}
+	// receiving as argument another ComplexNumber and doing the sum of the numbers
+	public static ComplexNumber sum(ComplexNumber num1, ComplexNumber num2) {
+		ComplexNumber sum = new ComplexNumber();
+		sum.real = num1.real + num2.real;
+		sum.im = num1.im + num2.im;
+		return sum;
+	}
+	// receiving as param another ComplexNumber and doing the product of the numbers
+	public static ComplexNumber product(ComplexNumber num1, ComplexNumber num2) {
+		ComplexNumber product = new ComplexNumber();
+		product.real = (num1.real*num2.real - num1.im*num2.im);
+		product.im = (num1.real*num2.im + num2.real*num1.im);
+		return product;
+	}
+	// printing the complex number in the format: a + bi (real + "+" + im + "*" + "i")
+	public String toString(ComplexNumber num1, ComplexNumber num2) {
```

Comment:
The toString method should display the current ComplexNumber. Delete the arguments num1, num2, the method does not need to receive any argument. Inside it change the line to String format = (this.real + "+" + this.im + "*" + "i");
"this" is a reference to the current object (the current complex number).